gen/flights/ai_flight_planner.py

```diff
@@ -179,10 +179,10 @@ def commision_cap(self):
                 orbit1p = loc.position.point_from_heading(hdg + 90, radius)
             else:
                 loc = self.from_cp.position.point_from_heading(random.randint(0, 360), random.randint(nm_to_meter(5), nm_to_meter(40)))
-                hdg = self.from_cp.position.heading_between_point(loc.position)
```

Comment:
"loc" in this else tree is already a "Point" containing only an X and Y coordinate and does not contain the value "position".